Makefile

```diff
@@ -1,56 +1,24 @@
-BIN := kube-mgmt
-PKG := github.com/open-policy-agent/kube-mgmt
-REGISTRY ?= openpolicyagent
-VERSION := 0.12.1
-ARCH := amd64
-COMMIT := $(shell ./build/get-build-commit.sh)
-
-IMAGE := $(REGISTRY)/$(BIN)
-
-BUILD_IMAGE ?= golang:1.12-alpine
-
-.PHONY: all
-all: image
+IMAGE := openpolicyagent/kube-mgmt
+VERSION ?= $(shell git describe --tags --dirty | sed 's/^v//g')
+COMMIT := $(shell git rev-parse --short HEAD)
 
 .PHONY: build
 build:
-	docker run \
-		-v $$(pwd)/.go:/go \
-		-v $$(pwd):/go/src/$(PKG) \
-		-v $$(pwd)/bin/linux_$(ARCH):/go/bin \
-		-v $$(pwd)/.go/std/$(ARCH):/usr/local/go/pkg/linux_$(ARCH)_static \
-		-w /go/src/$(PKG) \
-		$(BUILD_IMAGE) \
-		/bin/sh -c "ARCH=$(ARCH) VERSION=$(VERSION) COMMIT=$(COMMIT) PKG=$(PKG) ./build/build.sh"
-
-.PHONY: image
-image: build
-	docker build -t $(IMAGE):$(VERSION) -f Dockerfile .
-	docker tag $(IMAGE):$(VERSION) $(IMAGE):latest
+	docker build --build-arg VERSION=$(VERSION) --build-arg COMMIT=$(COMMIT) -t $(IMAGE):$(VERSION) .
+	helm package charts/opa --version $(VERSION) --app-version $(VERSION)
 
 .PHONY: clean
 clean:
-	rm -fr bin .go
+	docker rmi -f $(IMAGE):$(VERSION) 
+	rm -f *.tgz
 
-.PHONY: undeploy
-undeploy:
-	kubectl delete -f manifests/deployment-opa.yml || true
-
-.PHONY: deploy
-deploy:
-	kubectl create -f manifests/deployment-opa.yml
-
-.PHONY: up
-up: image undeploy deploy
-
-.PHONY: push
-push:
+.PHONY: docker-push
+docker-push:
 	docker push $(IMAGE):$(VERSION)
-
-.PHONY: push-latest
-push-latest:
+	docker tag $(IMAGE):$(VERSION) $(IMAGE):latest
```

Comment:
I think it's better if the `push` target pushes only the released version. We could add a `tag-latest` and `push-latest` target to add the `latest` tag and push it.

Reply:
This doesnt make sense in this PR
Latest is only pushed on release

Reply:
What I mean is we keep the following targets `push`, `push-latest` and `tag-latest`. Then in the release workflow we can call
```
make build
make push
make tag-latest
make push-latest
```